tools/distribution/src/release/assets/gh_asset.py

```diff
@@ -136,7 +136,7 @@ def validate(self, zip_directory: DirectoryMatcher, in_version: Version) -> bool
 
         dir = zip_directory.get('CrashReporter.xcframework')
         dir.assert_it_has_files([
-            'ios-arm64_arm64e_armv7_armv7s',
+            'ios-arm64_arm64e',
             'ios-arm64_i386_x86_64-simulator',
         ])
```

Comment:
Oh, I think we have to use `in_version` check to only validate this in and above `1.12.1`, no?
```python
if in_version.is_newer_than_or_equal('1.12.1'):
     dir.assert_it_has_files([
          'ios-arm64_arm64e',
          'ios-arm64_i386_x86_64-simulator',
     ])
else:
     dir.assert_it_has_files([
          'ios-arm64_arm64e_armv7_armv7s',
          'ios-arm64_i386_x86_64-simulator',
     ])
```
Otherwise we won't be able to resubmit XCFrameworks for previous tags.

Reply:
Ah yes indeed! Thanks!!